### ISSUE TITLE
Support for platformio?

### ISSUE THREAD

**Issue by jarzebski** [mattairtech/ArduinoCore-samd#2]
Hi, did you plan support platformIO ? Will be great!  http://platformio.org/boards ?

**Comment by mattairtech** [comment]
Yes, I will support PlatformIO, though I won't have time for this until after I release the next arduino core sometime in January.

**Comment by joseangeljimenez** [comment]
I am willing to contribute/help you to support platformIO for your boards definition. In case you are OK with it... How can I help you?

**Comment by glennsky** [comment]
While for platformio & CAN bus support for the SAMC21 - I stumbled upon this repository:
[https://github.com/hugllc/samc21_can](url) 

Which has a fork of the platformio platform-atmelsam that works with SAMC21.
[https://github.com/prices/platform-atmelsam](url)
and a fork of ArduinoCore-samd
[https://github.com/prices/ArduinoCore-samd](url)

Perhaps this could help with getting the mattairtech ArduinoCore-samd working with platformio?!?

**Comment by novvere** [comment]
Any advance on this enhancement? I would also like to help if needed.
